src/logger.cpp

```diff
@@ -16,7 +16,7 @@ namespace octo::logger
 {
 void Logger::dump_log(const Log& log) const
 {
-    Manager::instance().dump(log, channel_view_.channel().channel_name(), context_info_);
+    Manager::instance().dump(log, channel_view_.channel().channel_name(), log.context_info() + context_info_);
```

Comment:
This will cause a dropdown in performance
why not put the context info on a channel instead of each log?
or why not pass both the local and global context info's as references, and just use them on hte dump itself

Reply:
not more copies, verified in testss